.github/workflows/run-tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
             fail-fast: true
             matrix:
                 os: [ubuntu-latest]
-                php: [8.1, 8.0, 7.4]
+                php: [8.2, 8.1, 8.0, 7.4]
```

Comment:
```suggestion
                php: ['8.2', '8.1', '8.0', '7.4']
```

Reply:
@sanmai Most of the other Spatie workflows are not quoted, so just curious - why this one? :thinking:

Reply:
- `shivammathur/setup-php` [expects to see strings](https://github.com/shivammathur/setup-php#php-version-required) for `php-version`.
- There was an issue when it installed the really latest version if you didn't specify a string. I.e. for `8.0` it used to install `8.1.x.` and for `'8.0'` it used to install `8.0.x`. But it looks like it is no longer an issue.
- Lastly, it looks prettier in the log: